docs/source/ops/connect/helm.rst

```diff
@@ -6,7 +6,7 @@
 Daml Helm Chart
 ===============
 
-As of 1.18.0, we provide an Early Access version of the Helm Chart for
+We provide an Early Access version of the Helm Chart for
```

Comment:
Feel free to change this, but we'll soon remove this page (temporary measure until the Helm Chart is restored).